solutions/secure-hybrid-network/nestedtemplates/mock-onprem-azuredeploy.bicep

```diff
@@ -61,15 +61,19 @@ resource mocOnpremNetworkResource 'Microsoft.Network/virtualNetworks@2023-04-01'
   }
 }
 
-resource mocOnpremGateway_publicIPAddress 'Microsoft.Network/publicIPAddresses@2023-04-01' = if (configureSitetosite) {
+resource mocOnpremGateway_publicIPAddress 'Microsoft.Network/publicIPAddresses@2023-11-01' = if (configureSitetosite) {
   name: mocOnpremGateway.publicIPAddressName
   location: location
+  sku: {
+    name: 'Standard'
+    tier: 'Regional'
+  }
   properties: {
-    publicIPAllocationMethod: 'Dynamic'
+    publicIPAllocationMethod: 'Static'
```

Comment:
Standard sku publicIp /subscriptions/9c257b18-344d-4d4b-b09e-5c0a80b50e48/resourceGroups/rg-site-to-site-mock-prem-eastus2/providers/Microsoft.Network/publicIPAddresses/pip-onprem-vpn-gateway must have AllocationMethod set to Static. (Code: StandardSkuPublicIPAddressesMustBeStatic)